gunicorn/config.py

```diff
@@ -208,6 +208,14 @@ def sendfile(self):
 
         return True
 
+    @property
+    def buf_read_size(self):
+        buf_read_size = self.settings['buf_read_size'].get()
+        if buf_read_size is None:
+            return 1024
+
+        return int(buf_read_size)
```

Comment:
i think we should add some exception handling here if buf_read_size is not convertable to int and return a explanatory error.